group-coordinator/src/main/java/org/apache/kafka/coordinator/group/consumer/TopicMetadata.java

```diff
@@ -40,23 +45,31 @@ public class TopicMetadata {
      */
     private final int numPartitions;
 
+    /**
+     * Map of every partition to a set of its rackIds.
+     * If the rack information is unavailable, this is an empty map.
```

Comment:
nit: Should we say that the map contains mapping for partitions only of the partitions has racks or something like this?

Reply:
changed the wording